src/containers/ConfirmPassword/confirmPass.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input, Icon, Button, Form } from 'antd';
+import { ReactComponent as ConfirmImg } from '../assets/images/confirmPass.svg';
+import Header from '../../components/Header';
+
+import './confirmPass.css';
+
+const confirmPass = props => {
+  const {
+    history: { goBack },
+    form: { getFieldDecorator, validateFieldsAndScroll },
+  } = props;
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    validateFieldsAndScroll((err, values) => {
```

Comment:
You don't need `validateFieldsAndScroll` because it's only one input in the page so you can use `validateFields`
```suggestion
    validateFields((err, values) => {
```
It does the same job but without the scroll thing